app/services/verkiezing.js

```diff
@@ -6,6 +6,12 @@ export default class VerkiezingService extends Service {
   @service store;
 
   async checkIfPersonIsElected(personId, bestuursorgaan) {
+    const verkiezing = await bestuursorgaan.verkiezing;
```

Comment:
```suggestion
    const verkiezing = await bestuursorgaan?.verkiezing;
```

Zo dat het niets breekt? als er geen bestuursorgaan zou zijn